source/extensions/config_subscription/grpc/new_grpc_mux_impl.cc

```diff
@@ -328,5 +328,27 @@ absl::optional<std::string> NewGrpcMuxImpl::whoWantsToSendDiscoveryRequest() {
   return absl::nullopt;
 }
 
+class NewGrpcMuxFactory : public MuxFactory {
```

Comment:
nit: brief class comment, please. (What does "New" mean here?)

Reply:
That's unfortunate naming scheme that was introduced with Delta-xDS a while ago (new=delta, non-new is SotW).
While we are trying to move to a new (unified) code base, there are a few kinks due to small differences in behaviors of different xDS-servers that we need to iron out.